test/server/cards/01_SOR/units/FirstLegionSnowTrooper.spec.js

```diff
@@ -0,0 +1,86 @@
+describe('First Legion Snow Trooper', function() {
+    integration(function() {
+        describe('First Legion Snow Trooper while attacking non-damaged unit.', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['first-legion-snowtrooper'],
+                    },
+                    player2: {
+                        groundArena: ['yoda#old-master'],
+                    }
+                });
+            });
+
+            it(' Ability should do nothing', function () {
```

Comment:
So the jasmine test framework concatenates the test strings together hierarchically to make the test name. With what you have right now the test name would be `First Legion Snow Trooper - integration - First Legion Snow Trooper while attacking non-damaged unit.  Ability should do nothing`

Typically we structure the names to be more like `First Legion Snow Trooper - integration - Snow Trooper's ability should have no effect when attacking a non-damaged unit`. Ditto for all test names throughout

Reply:
I fixed the texts to adhere to the standard. Thank you for the explanation.